MODULE.bazel

```diff
@@ -0,0 +1,18 @@
+module(
+    name = "rules_cuda",
+    compatibility_level = 1,
+    version = "0.1.1",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.2.1")
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "rules_cc", version = "0.0.2")
```

Comment:
Is rules_cc actively maintained? https://github.com/bazelbuild/rules_cc/issues/86

Reply:
AFAIK adding a bazel_dep on `rules_cc` is how you express to bzlmod the fact that you are using cc rules in your module.  e.g. https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/abseil-cpp/20211102.0/MODULE.bazel

The convention seems to be that all modules using cc rules are adding a dep on rules_cc as a module.

Reply:
This is weird, because we don't actually load anything from `@rules_cc`. Can we leave some comment about the weirdness here.

@meteorcloudy @Wyverald Would you mind to @ someone that is familiar with `rules_cc` to make it clear of the current state of it and correct practice of depending on bazel builtin rules?

Reply:
hmm? if you don't use rules_cc, you don't need to depend on it.

It's also in a bit of a weird state right now since we modularized it, but most of the actual logic is still shipped with the Bazel binary, so the dependency is a bit ... cosmetic. @oquenchil can answer question in detail.

Reply:
I guess the question is one of forward compatibility - if the plan is to remove the builtin `cc_*` targets and require them being loaded from rules_cc in the future then it is more future compatible to be loading them now even though they just forward through to the builtin rules as it won't break in a future version of bazel (and thus break the BCR entry for a newer bazel)

Reply:
@jsharpe It would break anyway as the pinned version of `rules_cc` still just delegates to the native rules. Supporting a future version of Bazel without these native rules would at least require a version update. At that point maintaining forward compatibility becomes somewhat moot.

Reply:
Yes but it could be fixed fairly trivially by a bazel_dep in the root module to bump the version of `rules_cc` to one that fixes it for the version of bazel that removes the native rule?